hapi-fees/src/test/java/com/hedera/services/pricing/FeeSchedulesTest.java

```diff
@@ -64,38 +64,38 @@ void computesExpectedPriceForFeeScheduleUpdate() throws IOException {
 				"    \"hederaFunctionality\" : \"TokenFeeScheduleUpdate\",\n" +
 				"    \"fees\" : [ {\n" +
 				"      \"nodedata\" : {\n" +
-				"        \"constant\" : 74935946,\n" +
-				"        \"bpt\" : 119800,\n" +
-				"        \"vpt\" : 299500304,\n" +
+				"        \"constant\" : 74741325,\n" +
+				"        \"bpt\" : 119489,\n" +
+				"        \"vpt\" : 298722453,\n" +
```

Comment:
How did we come up with these changes? Any of the tests failed?

Reply:
@qnswirlds  Checked with Michael regarding this test. The test is actually failing on `01722-M-CreateTokenFeeSchUpdOp` this branch. Running `mvn test` all other tests passed.

Reply:
Sounds good to me. Thanks!